tests/Doctrine/Tests/Persistence/Mapping/AnnotationDriverTest.php

```diff
@@ -20,6 +24,15 @@ public function testGetAllClassNames(): void
 
         self::assertSame([TestClass::class], $classes);
     }
+
+    /**
+     * @return Generator<string, array{string}>
+     */
+    public function pathProvider(): Generator
+    {
+        yield 'straigthforward path' => [__DIR__ . '/_files/annotation'];
+        yield 'winding path' => [__DIR__ . '/../_files/annotation'];
```

Comment:
```suggestion
        yield 'winding path' => [__DIR__ . '/../Mapping/_files/annotation'];
```

Reply:
Oh right, I meant to type this actually :sweat_smile: